package.json

```diff
@@ -31,5 +32,11 @@
   "license": "MIT",
   "workspaces": [
     "packages/*"
-  ]
+  ],
+  "resolutions": {
+    "jstransformer-markdown-it": "2.0.0",
+    "mkdirp": "0.5.1",
+    "shallow-clone": "2.0.0",
+    "stylus": "0.54.5"
+  }
```

Comment:
These might cause problems - users of pug will not have these resolutions.

Reply:
dropped.